samples/Yacc/words2.yy

```diff
@@ -0,0 +1,53 @@
+/* File taken from http://home.adelphi.edu/~sbloch/class/271/examples/lexyacc/words2.y . */
```

Comment:
Couldn't find the license for this. Is it indicated anywhere on the website?

Reply:
Me neither, I find it very hard to find an example file for the .y files with a license - Since it was published like this online on an .edu site I was assuming. As we all know, assumptions are the mother of all f*ckups, so I'll have to find something else for this.

The GMX files are my own, I'll add a MIT license to the repository there.